.gitlab/functional_test/system_probe.yml

```diff
@@ -37,12 +37,49 @@
   - $S3_CP_CMD $S3_ARTIFACTS_URI/minimized-btfs-${ARCH}.tar.xz /tmp/minimized-btfs.tar.xz
   - cp /tmp/minimized-btfs.tar.xz $DD_AGENT_TESTING_DIR/site-cookbooks/dd-system-probe-check/files/minimized-btfs.tar.xz
 
+# The pull_test_dockers* jobs are duplicated here because we want the functional_tests stage to start at the same time as kernel_matrix_testing stage.
+# The ebpf-platform team is trying to measure the time from the start of the pipeline to the completion of the kernel_matrix_testing and functional_tests stages, to measure improvement.
+# The CI visibility product currently does not provide a way to measure this in the datadog app.
+# We are trying to hack around this by using the pull_test_dockers* jobs to trigger the stages at the start of the pipelines. 
+# This way the stage.duration metric will allow us to measure what we want.
+# This will most likely be temporary, until we decide which approach to move forward with for testing in the future.
+.pull_test_dockers_ft:
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/docker_x64$DATADOG_AGENT_BUILDIMAGES_SUFFIX:$DATADOG_AGENT_BUILDIMAGES
+  needs: []
+  tags: ["runner:docker"]
+  stage: functional_test
+  rules:
+    !reference [ .on_system_probe_changes_or_manual ]
+  script:
+    # DockerHub login for build to limit rate limit when pulling base images
+    - DOCKER_REGISTRY_LOGIN=$(aws ssm get-parameter --region us-east-1 --name ci.datadog-agent.$DOCKER_REGISTRY_LOGIN_SSM_KEY --with-decryption --query "Parameter.Value" --out text)
+    - aws ssm get-parameter --region us-east-1 --name ci.datadog-agent.$DOCKER_REGISTRY_PWD_SSM_KEY --with-decryption --query "Parameter.Value" --out text | docker login --username "$DOCKER_REGISTRY_LOGIN" --password-stdin "$DOCKER_REGISTRY_URL"
+    # Pull base images
+    - mkdir $KITCHEN_DOCKERS
+    - inv -e system-probe.save-test-dockers --output-dir $KITCHEN_DOCKERS --arch $ARCH
```

Comment:
One thing we could do is, instead of duplicated the pull_test_dockers* jobs, we could just add a dummy job in functional tests to trigger the start of the stage.

Reply:
I'd prefer a dummy job over duplicates.